config/ie/options

```diff
@@ -249,12 +241,18 @@ GAFFERCORTEX = True
 # get include locations right
 ##########################################################################
 
+alembicVersion = targetAppReg.get( "AlembicVersion", cortexReg["AlembicVersion"] )
 boostVersion = targetAppReg.get( "boostVersion", cortexReg["boostVersion"] )
-tbbVersion = targetAppReg.get( "tbbVersion", cortexReg["tbbVersion"] )
+ocioVersion = targetAppReg.get( "OpenColorIOVersion", cortexReg.get( "OpenColorIOVersion" ) )
```

Comment:
If `cortexReg.get()` returns `None`, won't the `os.path.join()` on L261 fail?

Reply:
According to the comment ( L135 ) the "last" fallback ( reading the registry of Gaffer for `OpenColorIO`  ) can be removed once all shows have updated to Cortex 10.1+.

Looking into more details at the registry, `OpenColorIO` is not part of the registry for Gaffer for recent version anyways ( > 0.58 ).

I believe this is a requirement for Cortex to specify an `OpenColorIOVersion` since version 10 so I think it should be fine in practice.

Reply:
> I believe this is a requirement for Cortex to specify an OpenColorIOVersion since version 10 so I think it should be fine in practice.

In that case maybe `cortexReg.get["OpenColorIOVersion"]` is more appropriate?

Reply:
Sorry I misinterpreted your comment. I updated the PR with your suggestion.